CHANGELOG/CHANGELOG-1.11.0.md

```diff
@@ -0,0 +1,76 @@
+<!--toc-->
+- [v1.11.0](#v1110)
+  - [Changelog since v1.10.2](#changelog-since-v1102)
+  - [Known Issues](#known-issues)
+  - [Changes by Kind](#changes-by-kind)
+    - [Deprecation](#deprecation)
+    - [Features](#features)
+    - [Bugs](#bugs)
+ 
+
+# v1.11.0 
+
+## Changelog since v1.10.2 
+
+## Known Issues 
+
+Controller publish is taking too long to complete/ Health monitoring is causing the Unity array to panic by opening multiple sessions. As a workaround, Disable Volume health monitoring on the node and keep it only at the controller level. Refer [here](https://dell.github.io/csm-docs/docs/csidriver/features/unity/#volume-health-monitoring) for more information about enabling/disabling volume health monitoring.
```

Comment:
"... As a workaround, Disable Volume health... " - Disable should be disable and Volume should be volume.  What other information can be provided besides "Controller publish is taking too long?".  What about things that would be observed in the logs? Under what circumstances would this be seen?

Reply:
Addressed the case sensitive issues, updated the k8s error log.